packages/chaire-lib-frontend/src/components/dashboard/SplitView.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { createRef, useEffect, useState } from 'react';
```

Comment:
Copyright header is missing in this new file, see any other code file for examples